### DISCUSSION TITLE
Increase bus factor

### DISCUSSION THREAD

**Discussion by Freed-Wu** [termux/termux-language-server#8] [Announcements]
Thank @Grimler91 for hosting this repo here.

Is there some one who have accounts of <https://readthedocs.org>, <https://pypi.org/> or <https://aur.archlinux.org/> and want to be added to maintainers by those accounts? It should can increase [bus factor](https://en.wikipedia.org/wiki/Bus_factor).

**Comment by thunder-coding** [comment]
`@CodingThunder` on AUR is me. I dont have accounts on pypi and readthedocs.

**Comment by Freed-Wu** [comment]
Done :+1:

**Comment by Grimler91** [comment]
Hey, please add/invite "grimler" on pypi and AUR, thanks!

**Comment by Freed-Wu** [comment]
Done :+1:
